docs/Other SDL Features/Batch Sending RPCs/index.md

```diff
@@ -58,3 +106,44 @@ sdlManager.sendSequential(requests: [createInteractionChoiceSet, performInteract
     print("All requests sent \(success ? "successfully" : "unsuccessfully")")
 }
 ```
+!@
+
+@![android,javaSE,javaEE]
+`// TODO maybe update the example to match iOS example. The texts states using createInteractionChoiceSet and performInteraction but shows an example of button subscribing. The choice menu manager is nearly done for Android so it would be nice if the code examples matched for better alignment.`
```

Comment:
Just a note: this TODO is still correct. The above example of subscribing buttons is a good example for the concurrent use case, while the choice sets are a better example for the sequential use case.

Reply:
kept